Gems/RemoteTools/gem.json

```diff
@@ -0,0 +1,23 @@
+{
+    "gem_name": "RemoteTools",
+    "display_name": "Remote Tools Connection",
```

Comment:
should this (and `summary`) just say "Remote Tools" now?

Reply:
I actually thought Remote Tools Connection was a bit more descriptive.